### PR TITLE
Use golang.org/x/sys/unix instead of syscall for Utimes()

### PR DESCRIPTION
**Pull request by mvo5**
[PLEASE DO NOT MERGE BEFORE WE HAVE A WORKING ARMHF BUILD AGAIN]

Lets see if x/sys/unix  actually works on armhf/arm64 and Utimes().

### CODE REVIEW

File: helpers/cp_test.go [hunk 1]
```diff
@@ -223,12 +224,10 @@ func (s *cpSuite) TestCopyPreserveAll(c *C) {
...
 	// Give the file a different mtime to ensure CopyFlagPreserveAll
 	// really works.
-	//
-	// You wonder why "touch" is used? And want to me about
-	// syscall.Utime()? Well, syscall not implemented on armhf
-	// Aha, syscall.Utimes() then? No, not implemented on arm64
-	// Really, this is a just a test, touch is good enough!
-	err = exec.Command("touch", src, "-d", "2007-08-23 08:21:42").Run()
+	err = unix.Utimes(src, []unix.Timeval{
+		unix.Timeval{Sec: 0, Usec: 0},
```
Comment (niemeyer) [inline helpers/cp_test.go]: On slices, there's no need to repeat the type for every element. E.g.: `[]T{T{}, T{}, T{}} == []T{{},{},{}}`

**Comment by niemeyer** [comment]
THIS BRANCH LGTM, BUT I WON'T MERGE IT.

File: helpers/cp_test.go (hunk 1 above)
Comment (mvo5) [inline helpers/cp_test.go]: Nice, I was not aware of this, thats cool

**Comment by mvo5** [comment]
Thanks Gustavo! Sorry for shouting and I reallyliked your reply :)

File: helpers/cp_test.go (hunk 1 above)
Comment (chipaca) [inline helpers/cp_test.go]: [Also true in map literals from 1.5, fwiw](https://golang.org/doc/go1.5#map_literals).

**Comment by chipaca** [comment]
any news on this?

**Comment by chipaca** [comment]
Have a second :+1: just in case you wanted it.

**Pull request merged**